read_vacuum.py

```diff
@@ -183,19 +188,39 @@ def __str__(self):
     signal.signal(signal.SIGQUIT, handleExit)
     signal.signal(signal.SIGTERM, handleExit)
 
+    sys.stdout.write("VACUUM READER\n***********\nPressure data will be live-plotted. Please save the plot manually before exiting vacuum_reader\n**********\n")
     reader = setUp(int(sys.argv[1]))
-    delaytime = 10.0 # inter-measurement delay time
+    delaytime = 9.0 # inter-measurement delay time
     try:
         # start data collection
+        timeAxis = []
+        piraniVals = []
+        capVals_1 = []
+        capVals_2 = []
+        figure = plt.figure(1)
+        plt.ion()
+        plt.show()
+
         while True:
             pirani_val = reader.pirani.getPressure()[0]
             capacitance_val = reader.capacitance.getPressure()
-            ostr = "%s\t%d\t%.02e\t%.02e\t%.02e\n" % (reader.isonow(), reader.timeElapsed(), pirani_val, capacitance_val[0], capacitance_val[1])
+            timeT = reader.timeElapsed()
+            ostr = "%s\t%d\t%.02e\t%.02e\t%.02e\n" % (reader.isonow(), timeT, pirani_val, capacitance_val[0], capacitance_val[1])
             reader.teeWrite(ostr)
-            if select.select([sys.stdin],[],[],delaytime)[0]:
-                readlines(sys.stdin)
-            else:
-                continue
```

Comment:
@krlynch I moved the delay into pausing on the graph, so the user could interact with it more easily. But, what was the reason for this use of select? Just want to be sure I didn't remove something important